modules/ssh/ssh.go

```diff
@@ -16,6 +22,52 @@ type Host struct {
 	SshKeyPair  *KeyPair
 }
 
+// ScpFileToE uploads the contents using SCP to the given host and fails the test if the connection fails.
+func ScpFileTo(t *testing.T, host Host, mode os.FileMode, remotePath, contents string) {
+	err := ScpFileToE(t, host, mode, remotePath, contents)
+	if err != nil {
+		t.Fatal(err)
+	}
+}
+
+// ScpFileToE uploads the contents using SCP to the given host and return an error if the process fails.
+func ScpFileToE(t *testing.T, host Host, mode os.FileMode, remotePath, contents string) error {
+	authMethods, err := createAuthMethodsForHost(host)
+	if err != nil {
+		return err
+	}
+	file := path.Base(remotePath)
+	dir := path.Dir(remotePath)
```

Comment:
Shouldn't this be `filepath` instead of `path`?

Reply:
Changed